Source/Widgets/Cesium3DTilesInspector/Cesium3DTilesInspectorViewModel.js

```diff
@@ -711,6 +737,15 @@ define([
     });
 
     /**
+     * Returns the url to the feature
+     * @param {Cesium3DTileFeature} feature The feature
+     * @returns {String} The url to that feature's tile
+     */
+    Cesium3DTilesInspectorViewModel.prototype.getFeatureUrl = function(feature) {
```

Comment:
Is there a reason we are doing this here rather than exposing `url` on the feature? (Or maybe even exposing `content` on the feature and then `url` on the content?  `url` on the feature seems like the minimal desired change here.